package.json

```diff
@@ -12,8 +12,9 @@
     "image:build": "./scripts/build-image.sh",
     "image:push": "./scripts/push-image.sh",
     "clean": "yarn rimraf dist",
-    "lint": "yarn eslint . --ext .js,.jsx,.ts,.tsx",
-    "lint:fix": "yarn eslint . --ext .js,.jsx,.ts,.tsx --fix ",
+    "lint": "yarn eslint . --ext .js,.jsx,.ts,.tsx && yarn prettier ./src --check",
+    "lint:fix": "yarn eslint . --ext .js,.jsx,.ts,.tsx --fix",
+    "prettier:fix": "yarn prettier ./src/ --write",
```

Comment:
I would say if checking the SCSS formatting is part of `yarn lint`, fixing it should be part of `yarn lint:fix`. I'm fine with having `prettier:fix` be in here too in case you want to run just prettier without the rest of eslint while you're developing (it's faster), but we should probably add `&& yarn prettier:fix` to the end of `lint:fix`.

Reply:
Same as above I replaced that.